### PR TITLE
[Keymap] Add Zoom keymap for ANAVI Macro Pad 8

### PR DESCRIPTION
**Pull request by leon-anavi**
Zoom keymap for ANAVI Macro Pad 8 with the following shortcuts:

* Alt+V: Start/stop video
* Alt+A: Mute/unmute my audio
* Alt+M: Mute/unmute audio for everyone except the host
* Alt+S: Start/stop screen sharing
* Alt+R: Start/stop local recording
* Alt+P: Pause/resume recording
* Alt+C: Start/stop cloud recording
* Switch to 2nd layout to control backlighting and underlighting

Signed-off-by: Leon Anavi <leon@anavi.org>

## Description

Add Zoom keymap for ANAVI Macro Pad 8 with the following shortcuts:

* Alt+V: Start/stop video
* Alt+A: Mute/unmute my audio
* Alt+M: Mute/unmute audio for everyone except the host
* Alt+S: Start/stop screen sharing
* Alt+R: Start/stop local recording
* Alt+P: Pause/resume recording
* Alt+C: Start/stop cloud recording
* Switch to 2nd layout to control backlighting and underlighting

This is keymap is convenient during video conference calls using the [popular software Zoom and implements its shortcuts](https://support.zoom.us/hc/en-us/articles/205683899-hot-keys-and-keyboard-for-zoom). 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/anavi/macropad8/keymaps/zoom/keymap.c [hunk 1]
```diff
@@ -0,0 +1,80 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/anavi/macropad8/keymaps/zoom/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/anavi/macropad8/keymaps/zoom/keymap.c (hunk 1 above)
Comment (leon-anavi) [reply keyboards/anavi/macropad8/keymaps/zoom/keymap.c:17]: Thank you for the feedback. I have just forced push an updated version of the patch a GPL2+ compatible license header based on the template you shared.

Thanks,
Leon

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Comment by leon-anavi** [comment]
Thank you very much for your time and code reviews. Probably you hear it every day but I just want to mention it again: QMK is an amazing open source project :)

Thanks,
Leon

**Pull request merged**